subsys/bluetooth/controller/ll_sw/ull.c

```diff
@@ -1226,6 +1226,47 @@ void *ull_disable_mark_get(void)
 	return mark_get(mark_disable);
 }
 
+/**
+ * @brief Stops a specified ticker using the ull_disable_(un)mark functions.
+ *
+ * @param ticker_handle The handle of the ticker.
+ * @param param         The object to mark.
+ * @param lll_disable   Optional object when calling @ref ull_disable
+ *
+ * @return 0 if success, else error.
+ */
+uint32_t ull_ticker_stop_with_mark(uint8_t ticker_handle, void *param,
```

Comment:
```suggestion
int ull_ticker_stop_with_mark(uint8_t ticker_handle, void *param,
```

Remove LL_ASSERTs, have them in respective callers by checking errno values returned from here (define at your descretion the errnos you want to convey to the caller) , helps debugging when assertion fails.

Reply:
Good idea! I've attempted to do so, although I'm not 100% sure on the returned ERRNO codes. Please review.